views/styles/chart_styles.js

```diff
@@ -100,4 +100,38 @@ export const chartStyles = (theme) => ({
   publicText: {
     alignSelf: 'center',
   },
+  tooltipContainer: {
+    width: '14vw',
+    padding: '1vw 1vw .5vw 1vw ',
+    background: 'rgba(255, 255, 255, 0.9)',
+  },
+  tooltipHeaderRow: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    width: '100%',
+    paddingBottom: '2%',
+    borderBottom: 'solid gray 1px',
+  },
+  tooltipContentRow: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    width: '100%',
+    paddingTop: '2%',
+    paddingBottom: '2%',
+  },
+  tooltipLabelColumn: {
+    display: 'flex',
+    flexDirection: 'column',
+    flexBasis: '40%',
+    flex: '1',
```

Comment:
if we make this `flex: '2'` and the value column `flex: '3'` can we remove the `flexBasis`?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/191/commits/82a8267770a707eedb1efe2f1cc9193d667b1d12